hw/ip/otbn/dv/otbnsim/sim/sim.py

```diff
@@ -6,34 +6,37 @@
 
 from .isa import OTBNInsn
 from .state import OTBNState
+from .stats import ExecutionStats
 from .trace import Trace
 
 
 class OTBNSim:
     def __init__(self) -> None:
         self.state = OTBNState()
         self.program = []  # type: List[OTBNInsn]
+        self.stats = ExecutionStats()
 
     def load_program(self, program: List[OTBNInsn]) -> None:
         self.program = program.copy()
 
     def load_data(self, data: bytes) -> None:
         self.state.dmem.load_le_words(data)
 
-    def run(self, verbose: bool) -> int:
+    def run(self, verbose: bool, collect_stats: bool) -> int:
         '''Run until ECALL.
 
         Return the number of cycles taken.
 
         '''
         insn_count = 0
         while self.state.running:
-            self.step(verbose)
+            self.step(verbose, collect_stats)
             insn_count += 1
 
         return insn_count
 
-    def step(self, verbose: bool) -> Tuple[Optional[OTBNInsn], List[Trace]]:
+    def step(self, verbose: bool,
```

Comment:
Nit: Could you add another newline before `verbose`? I think these things look better if either everything is one one line (which doesn't fit here) or each argument gets its own line.